recipes/lets-plot/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "lets-plot" %}
+{% set version = "4.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/JetBrains/lets-plot/archive/v{{ version }}.tar.gz
+  sha256: 25b743ea8de6bc4175d4eb6581881e31c2d8c723a143369d089c2a5178f09d0c
+
+build:
+  number: 0
+  script_env:
+   - PROJECT_VERSION=v{{ version }}
+
+requirements:
+  build:
+    - msys2::m2w64-gcc  # [win]
```

Please use the compiler directive and let the CI complete so that we can see the result.

```suggestion
    - {{ compiler('m2w64_cxx') }}  # [win]
```

---

[Fixed](https://github.com/conda-forge/staged-recipes/pull/27974/commits/c67436fc423fca478d523f8ee6dfce37b3743bc1).